lib/compat/wordpress-6.8/site-editor.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * Updates to the site editor in 6.8.
+ *
+ * Adds a mandatory dashboard link and redirects old urls.
+ *
+ * @package gutenberg
+ */
+
+add_filter(
+	'block_editor_settings_all',
+	function ( $settings ) {
+		$settings['__experimentalDashboardLink'] = admin_url();
```

Comment:
Note that when using `admin_url()` without arguments, it's recommended that we pass a `'/'` argument:

```suggestion
		$settings['__experimentalDashboardLink'] = admin_url( '/' );
```

While the result for the user is the same, we actually save an extra redirect. Otherwise, redirecting to `/wp-admin` will result in an one additional `301` to `/wp-admin/` - that's just how canonical redirects work on the admin side.